__tests__/index.test.js

```diff
@@ -16,8 +16,8 @@
 // const index = require('../index')
 
 test('that we can run tests', () => {
-  // your real tests go here
-  expect(1 + 2 + 3).toBe(6);
+    // your real tests go here
+    expect(1 + 2 + 3).toBe(6);
 });
 
 // For more information about testing with Jest see:
```

Comment:
I think we can delete this file as it is not being used.
Tests are written in the `/spec` directory.

Reply:
Done